lib/galaxy/config/sample/galaxy.yml.sample

```diff
@@ -20,16 +20,55 @@
 # ``uwsgi:`` section will be ignored if Galaxy is started via Gravity commands (e.g ``./run.sh``, ``galaxy`` or ``galaxyctl``).
 gravity:
 
+  # Process manager to use.
+  # ``supervisor`` is the default process manager when Gravity is invoked as a non-root user.
+  # ``systemd`` is the default when Gravity is invoked as root.
+  # Valid options are: supervisor, systemd
+  # process_manager:
+
+  # What command to write to the process manager configs
+  # `gravity` (`galaxyctl exec <service-name>`) is the default
+  # `direct` (each service's actual command) is also supported.
+  # Valid options are: gravity, direct, exec
```

Comment:
exec is not listed above, what does it do?

Reply:
It's not really an option, it's used internally to force the generation of actual command lines for `os.execvp()` when `galaxyctl exec ...` is called, but the [documentation generator]( https://github.com/galaxyproject/gravity/blob/13ce6c578b217b39e8933e98826a32cf10d7a8b3/gravity/util/__init__.py#L52) just spits out all possible values of the Enum. I'm not sure if there's a reasonable way to annotate it so that we can skip it? Maybe make the value `_exec` and skip any enum values that start with `_`?

Reply:
Yeah that worked. https://github.com/galaxyproject/gravity/pull/94